opennms-base-assembly/src/main/filtered/etc/events/NetgearProsafeSmartSwitch.events.xml

```diff
@@ -0,0 +1,389 @@
+<!-- Define traps sent by NetGear ProSafe Smart Switch line
+     {iso(1) identified-organization(3) dod(6) internet(1) private(4) enterprise(1) netgear(4526) ng700smartswitch(11) agentSwitching(1)
+     see NetGear-provided ng_switching.my MIB file (last updated 17 Mar 2008 12:00:00 GMT) included on switch CD
+-->
+
+<events xmlns="http://xmlns.opennms.org/xsd/eventconf">
+<event>
+	<mask>
+		<maskelement>
+			<mename>id</mename>
+			<mevalue>.1.3.6.1.4.1.4526.11.1</mevalue>
+		</maskelement>
+		<maskelement>
+			<mename>generic</mename>
+			<mevalue>6</mevalue>
+		</maskelement>
+		<maskelement>
+			<mename>specific</mename>
+			<mevalue>1</mevalue>
+		</maskelement>
+	</mask>
+	<uei>uei.opennms.org/vendor/netgear/traps/multipleUsersTrap</uei>
+	<event-label>NETGEAR-SWITCHING-MIB defined trap event: multipleUsersTrap</event-label>
+	<descr>
+&lt;p&gt;trapMgrMultipleUsersLogTrap.&lt;/p&gt;&lt;table&gt;&lt;/table&gt;
+	</descr>
+		<logmsg dest='logndisplay'>&lt;p&gt;
+			Netgear Event: multipleUsersTrap trap received&lt;/p&gt;
+		</logmsg>
+	<severity>Warning</severity>
+</event>
+<event>
+	<mask>
+		<maskelement>
+			<mename>id</mename>
+			<mevalue>.1.3.6.1.4.1.4526.11.1</mevalue>
+		</maskelement>
+		<maskelement>
+			<mename>generic</mename>
+			<mevalue>6</mevalue>
+		</maskelement>
+		<maskelement>
+			<mename>specific</mename>
+			<mevalue>2</mevalue>
+		</maskelement>
+	</mask>
+	<uei>uei.opennms.org/vendor/netgear/traps/broadcastStormStartTrap</uei>
+	<event-label>NETGEAR-SWITCHING-MIB defined trap event: broadcastStormStartTrap</event-label>
+	<descr>
+&lt;p&gt;trapMgrBCastStormStartLogTrap.&lt;/p&gt;&lt;table&gt;&lt;/table&gt;
+	</descr>
+		<logmsg dest='logndisplay'>&lt;p&gt;
+			Netgear Event: broadcastStormStartTrap trap received&lt;/p&gt;
+		</logmsg>
+	<severity>Warning</severity>
+</event>
+<event>
+	<mask>
+		<maskelement>
+			<mename>id</mename>
+			<mevalue>.1.3.6.1.4.1.4526.11.1</mevalue>
+		</maskelement>
+		<maskelement>
+			<mename>generic</mename>
+			<mevalue>6</mevalue>
+		</maskelement>
+		<maskelement>
+			<mename>specific</mename>
+			<mevalue>3</mevalue>
+		</maskelement>
+	</mask>
+	<uei>uei.opennms.org/vendor/netgear/traps/broadcastStormEndTrap</uei>
+	<event-label>NETGEAR-SWITCHING-MIB defined trap event: broadcastStormEndTrap</event-label>
+	<descr>
+&lt;p&gt;trapMgrBCastStormEndLogTrap.&lt;/p&gt;&lt;table&gt;&lt;/table&gt;
+	</descr>
+		<logmsg dest='logndisplay'>&lt;p&gt;
+			Netgear Event: broadcastStormEndTrap trap received&lt;/p&gt;
+		</logmsg>
+	<severity>Cleared</severity>
```

Comment:
"Cleared" is reserved for use with alarms and should never be used for the severity of individual events. Should be "Normal" instead.